underpants.js

```diff
@@ -192,6 +319,24 @@ var _ = {};
 }
 */
 
+// accepts array and a function
+// call function for for each elem
+// w/ arguments of elem, key, arr
+// return arr of 2 sub arrs:
+//    array that contains all values for truthy
+//    array that contains all values for falsy
+
+_.partition = (arr, func) => {
+    const newArr = [[], []];
+    _.each(arr, function (elem, index, arr2) {
+        if (func(elem, index, arr2)) {
+            newArr[0].push(elem);
+        } else {
+            newArr[1].push(elem);
+        }
+    });
+    return newArr;
```

Comment:
There is a pretty clean solution here that doesn't require you to call `_.each`.

I'd also argue for clarity's sake that this code would be more clear if you created two arrays (perhaps named `accepted` and `rejected`) to store your values, and then returned `[accepted, rejected]`. It's not immediately clear what the significance of newArr being a 2D array is.